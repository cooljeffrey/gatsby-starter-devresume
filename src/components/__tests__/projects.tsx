import React from "react";
import renderer from "react-test-renderer";
import Projects from "../projects";
describe("Projects", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(
        <Projects
          items={[
            {
              title: "project title",
              meta: "Opensource",
              summary: "long project summary",
            },
            {
              title: "project title 2",
              meta: "Opensource",
              summary: "long project summary",
            },
          ]}
        />
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
