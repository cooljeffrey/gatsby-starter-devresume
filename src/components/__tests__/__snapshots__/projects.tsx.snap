// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Projects renders correctly 1`] = `
<section
  className="project-section py-3"
>
  <h3
    className="text-uppercase resume-section-heading mb-4"
  >
    Projects
  </h3>
  <div
    className="item mb-3"
  >
    <div
      className="item-heading row align-items-center mb-2"
    >
      <h4
        className="item-title col-12 col-md-6 col-lg-8 mb-2 mb-md-0"
      >
        project title
      </h4>
      <div
        className="item-meta col-12 col-md-6 col-lg-4 text-muted text-left text-md-right"
      >
        Opensource
      </div>
    </div>
    <div
      className="item-content"
    >
      <p>
        long project summary
      </p>
    </div>
  </div>
  <div
    className="item mb-3"
  >
    <div
      className="item-heading row align-items-center mb-2"
    >
      <h4
        className="item-title col-12 col-md-6 col-lg-8 mb-2 mb-md-0"
      >
        project title 2
      </h4>
      <div
        className="item-meta col-12 col-md-6 col-lg-4 text-muted text-left text-md-right"
      >
        Opensource
      </div>
    </div>
    <div
      className="item-content"
    >
      <p>
        long project summary
      </p>
    </div>
  </div>
</section>
`;
